substreams-helper/src/price.rs

```diff
@@ -0,0 +1,52 @@
+use std::ops::Div;
+use std::str::FromStr;
+
+use bigdecimal::BigDecimal;
+use serde::Deserialize;
+use substreams::Hex;
+
+use crate::{math, rpc};
+
+pub enum Network {
+    Ethereum,
+}
+
+#[derive(Deserialize)]
+struct Config {
+    ethereum: NetworkConfig,
+}
+
+#[derive(Deserialize)]
```

Comment:
why does this need to be serializable?

Reply:
Gonna remove. It was there simply bc I used to read config from toml file. That way serializable is required